tests/test_active.py

```diff
@@ -82,20 +82,20 @@ def test_compute_03(self):
                                 [0.441965, -0.467131, -0.055723, 0.763774]])
         np.testing.assert_array_almost_equal(true_evects, ss.evects)
 
-    def test_compute_04(self):
-        np.random.seed(42)
-        gradients = np.random.uniform(-1, 1, 60).reshape(15, 4)
-        weights = np.ones((15, 1)) / 15
-        ss = ActiveSubspaces()
-        ss.compute(gradients=gradients,
-                   weights=weights,
-                   method='exact',
-                   nboot=250)
-        true_cov_matrix = np.array([[0.295783, 0.004661, 0.057825, -0.056819],
-                                    [0.004661, 0.427352, 0.086039, 0.160164],
-                                    [0.057825, 0.086039, 0.445253, -0.019482],
-                                    [-0.056819, 0.160164, -0.019482, 0.316237]])
-        np.testing.assert_array_almost_equal(true_cov_matrix, ss.cov_matrix)
+    # def test_compute_04(self):
```

Comment:
what is the problem here?

Reply:
I removed the covariance matrix since I don't save it in kas and I need the same outputs from the function _build_decompose_cov_matrix fot kas and as both. See line 60 of subspaces.py